doc/manual/install-domserver.rst

```diff
@@ -74,6 +78,12 @@ DOMjudge uses a MySQL or MariaDB database server for information storage.
 Where this document talks about MySQL, it can be understood to also apply
 to MariaDB.
 
+For the install option (unless using bare-install) steps you need the
+python `yaml` module, for Debian GNU/Linux, or one of its derivative 
+distributions like Ubuntu::
+
+  sudo apt install python3-yaml
+
```

Comment:
Let's just add this to the overall package install list. I don't think that's a big burden and it simplifies things.